test/xpu_api/numerics/numeric.ops/partial.sum/xpu_partial_sum.pass.cpp

```diff
@@ -53,6 +53,7 @@ template <class InIter, class OutIter, class Test> void test() {
   }
 }
 
+// TODO required to remove all these class names and use uniq_kernel_name
```

Comment:
We do really need such one minor comment for whole file? 
I think that keep that file unmodified is better

Reply:
Comment removed. No more changes in this file.